src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -66,6 +67,7 @@ Some commands take arguments. Pass no args or -h for usage.
   mapredcp        Dump CLASSPATH entries required by mapreduce
```

Comment:
branch-1 has this option after "mapredcp"

    completebulkload Run LoadIncrementalHFiles tool